src/common/root/index.js

```diff
@@ -1,10 +1,10 @@
 /* global fetch:false */
 import React, { useState, useEffect } from "react";
-import debounce from "lodash.debounce";
 import Linter from "../linter";
 import recommendedConfig from "stylelint-config-recommended";
 import standardConfig from "stylelint-config-standard";
 import "whatwg-fetch";
+import useDebouncedCallback from "use-debounce/lib/callback";
```

Comment:
I think better to use exported function than the internal directory path, so how about the following code? ⬇️ 

```suggestion
import { useDebouncedCallback } from "use-debounce";
```

There is not a usage of the exported function in `use-debounce` README, but actually it's available.
- https://github.com/xnimorz/use-debounce#debounced-callbacks
- https://github.com/xnimorz/use-debounce/blob/91a7fe01d6b321a2bbe8b5c8e80f4242564b15c3/src/index.js#L4

Reply:
I wanted to do this, but thought will follow readme :)